### PR TITLE
Add metadata to image alpha

### PR DESCRIPTION
**Pull request by Azaya89**
Added metadata to examples/plotting/image_alpha.py

- [x] issues: ref #11765 
- [x] tests added / passed

### CODE REVIEW

File: examples/plotting/image_alpha.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
+''' An example of plots showing how to add alpha value (transparency) to images for three different image methods.
```
Comment (bryevdv) [inline examples/plotting/image_alpha.py]:
```suggestion
''' An example demonstrating how to add alpha value (transparency) to images in different ways.
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks @Azaya89 just one suggested edit

File: examples/plotting/image_alpha.py (hunk 1 above)
Comment (Azaya89) [reply examples/plotting/image_alpha.py]: Done.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @Azaya89 !

**Pull request merged**